src/compiler/emitter.ts

```diff
@@ -2273,10 +2279,10 @@ namespace ts {
         function emitPropertyAccessExpression(node: PropertyAccessExpression) {
             const expression = cast(emitExpression(node.expression), isExpression);
             const token = getDotOrQuestionDotToken(node);
-            const indentBeforeDot = needsIndentation(node, node.expression, token);

```

Comment:
'indentation' in the old naming scheme seems to mean 'newline and indent', right?

Reply:
Yep, the only reason something “needed indentation” was if it was preceded by a newline.